### ISSUE TITLE
Add expand_keys type support for yaml files

### ISSUE THREAD

**Issue by stephenlawrence** [Cimpress-MCP/git2consul#54]
As a user, I would like to be able to import a .yml file and have git2consul split each key/value into k/v in Consul similar to how the json expand_keys works. This would allow me to standardize my configurations to yml.

**Comment by ryanbreen** [comment]
Sounds good.  I plan to do this within the scope of GH-49.

**Comment by syamsathyan** [comment]
Is there an ETA on this enhancement, thanks?

**Comment by spanktar** [comment]
Rather than an ETA, I'd be happy to know if it's being considered or worked on. If so, then I would know that eventually I can use it.  For now, I wrote my own janky yml->consul code.

**Comment by cleung2010** [comment]
@spanktar Yes, it is being considered. We are working refactor of git2consul to include file backends, but there is no ETA on this yet. Apologies for the delay!

**Comment by arthurtsang** [comment]
https://github.com/nodeca/js-yaml 
i've a project using ```js-yaml``` pretty easy to use.
